fw-child/resources/js/child-functions.js

```diff
@@ -142,6 +142,29 @@ const $ = jQuery;
       });
     }
 
+    // CARD LINKS HOVERING
+
+    function handleCardLinkHovering() {
+      if ( $( '.card.has-links' ).length ) {
+        $( '.card.has-links a.hover-toggle' ).hover(
+          function() {
+            $( this ).closest( '.card' ).addClass( 'card-hover' );
+          },
+          function() {
+            $( this ).closest( '.card' ).removeClass( 'card-hover' );
+          }
+        );
+      }
+    }
+
+    // Initial call on page load
+    handleCardLinkHovering();
```

Comment:
To prevent problems when cached JavaScript files end up being executed before the DOM has finished loading, it could be prefered to do:
```js
$( handleCardLinkHovering );
```

Reply:
@xfrenette-crim The function is already wrapped into:

```
(function ($) {
  $(function () {
    // Code here runs when the DOM is fully loaded
  });
})(jQuery);
```